### PR TITLE
DF-552:Documenting deployment steps to Github Public repo

### PR DESCRIPTION
**Pull request by TNA-Allan**
Ticket URL: https://national-archives.atlassian.net/browse/DF-552

## About these changes

Added deployment steps

## How to check these changes

At the time of deployment

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer.
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes.
- [x] Ensured that PR includes only commits relevant to the ticket.
- [x] Waited for all CI jobs to pass before requesting a review.
- [x] Added/updated tests and documentation where relevant.

## Merging a feature or bug fix?

1. Use the `Squash and merge` option to keep the target branch's commit history nice and clean.
2. Where relevant, include the ticket number in commit title, e.g. `DF-XXX: Ticket name / short description`.

## Merging a release branch?

1. Use the `Create a merge commit` option to preserve the original commit IDs.
2. Use the message format `release/<major.minor.patch>` when merging.

## Deployment guidance

1. Post to the appropriate Slack channel to check that it is okay to continue.
2. To start a deployment, push the relevant branch to the `platform` remote.
3. Update the appropriate Slack channel when the deployment is complete.
4. Update the status on the corresponding Jira ticket (where relevant).

### CODE REVIEW

**Comment by ababic** [comment]
@TNA-Allan  The heading structure feels a bit deep here. Maybe it would be simpler to just have `Pre-requisites for deployment` a `Deploying to develop` and `Deploying to production` as top-level headings?

File: docs/infra/environments.md [hunk 1]
```diff
@@ -1,9 +1,71 @@
...
+
+## How deployments are triggered
+
+- Deployments are manual at this time.
+
+### Pre-requisites
+
+- Requires user access on the hosted environment for deployments. Check with management if you need one.
+- Follow Branching Model and Naming branches [project-conventions] (<https://nationalarchives.github.io/ds-wagtail/developer-guide/project-conventions/>)
+- Check if it is Ok to deploy see guidance on deployment [.github\PULL_REQUEST_TEMPLATE.md](https://github.com/nationalarchives/ds-wagtail/blob/develop/.github/PULL_REQUEST_TEMPLATE.md)
```
Comment (ababic) [inline docs/infra/environments.md:18]: Actually, this page would be a better place all round for that guidance... maybe that should live here, and `PULL_REQUEST_TEMPLATE` should just contain a link to it?

File: docs/infra/environments.md [hunk 2]
```diff
@@ -1,9 +1,71 @@
...
+- Test on local before deploy to environment
+
+### main
+
+#### Set environment variables from the UI console for `main` environment
```
Comment (ababic) [inline docs/infra/environments.md:23]: If these are steps, it might be clearer to number these headings.

File: docs/infra/environments.md (hunk 1 above)
Comment (TNA-Allan) [reply docs/infra/environments.md]: Thanks, Done https://github.com/nationalarchives/ds-wagtail/pull/682/commits/04930722dfa5e3642430f999d51f2f6215422b09

File: docs/infra/environments.md (hunk 2 above)
Comment (TNA-Allan) [reply docs/infra/environments.md]: Thanks, Done https://github.com/nationalarchives/ds-wagtail/pull/682/commits/04930722dfa5e3642430f999d51f2f6215422b09

**Comment by TNA-Allan** [comment]
> @TNA-Allan The heading structure feels a bit deep here. Maybe it would be simpler to just have `Pre-requisites for deployment`, `Deploying to develop` and `Deploying to production` as h2s?

Thanks @ababic , Done.

**Pull request merged**